docs: remove NPM1k as per PR #154 (#185)
clearlydefined/clearlydefined: docs/resources/faq.md

Other technologies like SPDX, OpenChain, Quartermaster, etc. are
complementary to ClearlyDefined. Where they bring tools, process, approaches,
and data formats, ClearlyDefined brings the data itself. In this way ClearlyDefined is the

> proverbial _missing piece_. ClearlyDefined is most like [libraries.io](https://libraries.io)

Reviewer: ```suggestion
proverbial _missing piece_. ClearlyDefined is similar to [libraries.io](https://libraries.io)
```

> though we vary in breadth and depth.

Reviewer: I'm wondering what is the value in linking to libraries.io.
